src/agent/executor/base.ts

```
@@ -0,0 +1,87 @@
+        throw new Error(`Empty response for ${this.taskType}`);
+      }
+
+      return content;
+    } catch (error) {
+      throw error;
+    }
+  }
+
+  protected renderTemplate(template: string, context: AgentContext): string {
+    // Import here to avoid circular dependencies
+    const { renderTemplate } = require('@/utils/template');
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)